src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -472,6 +472,8 @@ public enum SslProtocols
         Tls = 192,
         Tls11 = 768,
         Tls12 = 3072,
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
The summary is that `XmlSerializer` will throw `System.InvalidOperationException` if an enum value with `ObsoleteAttribute` needs to be deserialzied. This is documented here:
https://msdn.microsoft.com/en-us/library/system.xml.serialization.xmlserializer(v=vs.105).aspx#Anchor_4 

@weshaggard Please follow-up internally for more details.